Detectors/TRD/base/src/PadPlane.cxx

```diff
@@ -158,3 +162,59 @@ void PadPlane::setNrows(int n)
   }
   mNrows = n;
 };
+
+double PadPlane::getPadRow(double z) const
+{
+  double lengthCorr = mLengthIPad / mLengthOPad;
```

Comment:
Since these methods are used a lot, this quantity can be stored as a class member when `mLengthIPad`, and `mLengthOPad` are defined.

Reply:
These variables are set in `Geometry.cxx` so I would have to set `mInverseLengthIPad` from there as well, correct?

Reply:
I wouldn't touch `Geometry.cxx`, but extend the setter in `PadPlane.h`:
```cpp
void setLengthIPad(double l) {
  mLengthIPad = l;
  mInverseLengthIPad = 1.0 / l;
}
```

Reply:
Oh I see, and then use multiplication with the new inverse member in `getPad()` and `getPadRow()`. I follow now.

Reply:
You can also store `mLengthCorr`, and anything that doesn't need to be recalculated after building the pad plane.

Reply:
@jolopezl hmm I would not know where to define `mLengthCorr`. To define this, one needs knowledge of both the `mWidthIPad` and `mWidthOPad`. Surely this would have to happen in the `Geometry` class?